cloud/blockstore/libs/storage/partition/part_actor_loadstate.cpp

```diff
@@ -228,7 +228,7 @@ void TPartitionActor::CompleteLoadState(
     Y_ABORT_UNLESS(State->GetGarbageQueue().AddNewBlobs(args.NewBlobs));
     Y_ABORT_UNLESS(State->GetGarbageQueue().AddGarbageBlobs(args.GarbageBlobs));
 
-    if (State->GetBaseDiskId()) {
+    if (State->GetBaseDiskId() && !partitionConfig.GetIsSystem()) {
```

Comment:
Need comment about why we disable logical used blocks calculation for "system" disks

Reply:
Still is not answered why we disable that for such disks

Reply:
And it would be better to explain what "proxy overlay" disks are used for